app/assets/stylesheets/concepts.scss

```diff
@@ -165,4 +165,9 @@ div.synonym-change-request button {
 #concepts_json_link{
```

Comment:
this one is used nowhere

Reply:
and is the same as your `.concepts_json_button`

Reply:
done in https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/649/commits/c77c20b7b1434a9cb85e28056f21e41d0830cb06